File: src/me/krickl/memebotj/InternalCommands/APIInformationCommand.java
package me.krickl.memebotj.InternalCommands;

import me.krickl.memebotj.ChannelHandler;
import me.krickl.memebotj.CommandHandler;
import me.krickl.memebotj.UserHandler;

public class APIInformationCommand extends CommandHandler {

	public APIInformationCommand(String channel, String command, String dbprefix) {
		super(channel, command, dbprefix);
		this.setNeededCommandPower(75);
	}
	
	@Override
	public void commandScript(UserHandler sender, ChannelHandler channelHandler, String[] data) {
		//this command will publicly print information about the api right now! use with caution.
		
		channelHandler.sendMessage("Current api connection: " + channelHandler.getApiConnectionIP() + " || Private Key for channel: " + channelHandler.getPrivateKey() + " || Private key for sender: " + sender.getPrivateKey(), this.getChannelOrigin());
	}
	
}

File: src/me/krickl/memebotj/InternalCommands/APIInformationCommand.scala
package me.krickl.memebotj.InternalCommands

import me.krickl.memebotj.ChannelHandler
import me.krickl.memebotj.CommandHandler
import me.krickl.memebotj.UserHandler

class APIInformationCommand(channel: String, command: String, dbprefix: String) extends CommandHandler(channel, command, dbprefix) {
  this.setNeededCommandPower(75)

  override def commandScript(sender: UserHandler, channelHandler: ChannelHandler, data: Array[String]) {
    channelHandler.sendMessage("Current api connection: " + channelHandler.getApiConnectionIP + " || Private Key for channel: " + channelHandler.getPrivateKey + " || Private key for sender: " + sender.getPrivateKey, this.getChannelOrigin)
  }
}